### DISCUSSION TITLE
How VLLM load Mistral Pixtral without Huggingface Config File?

### DISCUSSION THREAD

**Discussion by galaxy202411** [vllm-project/vllm#10187] [Q&A]
I loaded Mistral Pixtral by vllm successfully. It's great. 

However, it made me confused that pixtral in HF did not have any HF config file. Please refer to https://huggingface.co/mistralai/Pixtral-12B-2409/tree/main

How VLLM know the model type without any HF config file?

**Comment by DarkLight1337** [comment]
We know to load the repo in Mistral format via the CLI argument `--load-format mistral`.

**Reply by galaxy202411** [reply]
Thanks. I will have a try.
